corehq/apps/hqwebapp/password_validation.py

```diff
@@ -0,0 +1,34 @@
+from django.core.exceptions import ValidationError
+from django.utils.translation import ugettext as _
+
+from corehq.apps.hqwebapp.const import RESTRICT_USED_PASSWORDS_NUM
+from corehq.apps.hqwebapp.models import UsedPasswords
+from corehq.apps.hqwebapp.utils import verify_password, hash_password
+
+
+class UsedPasswordValidator(object):
+    def validate(self, password, user):
+        used_passwords = UsedPasswords.objects.filter(
+            user=user,
```

Comment:
The `UserPasswords` model has been removed and we're now storing the data in redis with a timeout.